### PR TITLE
add command to ping user when the build queue is empty

### PR DESCRIPTION
**Pull request by IanHibbert**
r? @jdm

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -455,6 +455,23 @@ var handlerWrapper = module.exports.handlerWrapper = function handlerWrapper(pin
...
+    if (message.indexOf('watch queue') > -1) {
+      if (!queueWatchers.includes(from)) {
+        queueWatchers.push(from);
+      }
+
+      bot.say(to, `i'll let you know when the queue is ready for you ${from}`);
+    }
+
+    if (message.indexOf('stop watching queue') > -1) {
+      const nickNameIndex = message.indexOf(from);
```
Comment (jdm) [inline server.js]: Shouldn't this be `queueWatchers.indexOf`?

Comment (IanHibbert) [reply server.js]: Yes, thanks for reviewing. I've made the change to the code.

**Comment by jdm** [comment]
This looks good, but it needs to be rebased against master because of merge conflicts.

**Comment by IanHibbert** [comment]
I rebased against master and got rid of the conflicts

**Comment by jdm** [comment]
Thanks!

**Pull request merged**